otterdog/adoptium.jsonnet

```diff
@@ -655,6 +655,23 @@ orgs.newOrg('adoptium') {
         orgs.newBranchProtectionRule('master'),
       ],
     },
+    orgs.newRepo('jdk22') {
```

Comment:
we created a template function for the mirrors so that they all have the same settings:

```
newMirrorRepo('jdk22') {},
```

should be used instead imho.